.github/dependabot.yml

```diff
@@ -2,37 +2,35 @@
 # https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
 version: 2
 updates:
-  # Dependencies listed in go.mod
-  - package-ecosystem: "gomod"
-    directory: "/" # Location of package manifests
-    schedule:
-      interval: "weekly"
-      day: "monday"      
-    commit-message:
-      include: "scope"
-    ignore:
-      # Ignore controller-runtime as its upgraded manually.
-      - dependency-name: "sigs.k8s.io/controller-runtime"
-      # Ignore k8s and its transitives modules as they are upgraded manually
-      # together with controller-runtime.
-      - dependency-name: "k8s.io/*"
-      - dependency-name: "go.etcd.io/*"
-      - dependency-name: "google.golang.org/grpc"
-    labels:
-      - "ok-to-test"
-      - "kind/cleanup"
-      - "priority/important-soon"
-      - "wg/security-audit"
-
-
-  # Dependencies listed in .github/workflows/*.yml
-  - package-ecosystem: "github-actions"
-    directory: "/"
-    schedule:
-      interval: "weekly"
-      day: "monday"
-    commit-message:
-      include: "scope"
-    labels:
-      - "kind/cleanup"
-      - "ok-to-test"
+# Dependencies listed in go.mod
+- package-ecosystem: "gomod"
+  directory: "/" # Location of package manifests
+  schedule:
+    interval: "weekly"
```

Comment:
/hold

maybe change this until we get it working for a quicker feedback loop?

Reply:
added via e0ec0e926f6bf969651685e2fff75a6f8ee394e6